src/main.cc

```diff
@@ -1,6 +1,51 @@
-#include "aqdata/aq_base.h"
+#ifdef TEST
+#include "gtest/gtest.h"
+#include "gmock/gmock.h"
+#include <unistd.h>
+#include <getopt.h>
+#include <deal.II/base/mpi.h>
 
-int main ()
-{
+#include "test_helpers/bart_test_helper.h"
+
+#endif
+
+//#include "aqdata/aq_base.h"
+
+#ifdef TEST
+int main(int argc, char* argv[]) {
+  // Parse optional arguments
+  ::testing::InitGoogleMock(&argc, argv);
+  
+  int option_index = 0;
+  
+  const struct option longopts[] =
+  {
+    {"report", no_argument, nullptr, 'r'}
+  };
+  
+  int c;
+  while ((c = getopt_long (argc, argv, "rd:", longopts, &option_index)) != -1) {
+    switch(c) {
+      case 'r':
+        btest::GlobalBartTestHelper().SetReport(true);
```

Comment:
I would suggest you change function and class naming with `Bart` to `BART` as each letter is actually the first letter of a word. Similar case is we have `ProduceAQ`

Reply:
I've updated that, as well as updating the class to `BARTTestHelper()`